AwsCryptographyPrimitives/src/AESEncryption.dfy

```diff
@@ -3,7 +3,7 @@
 
 include "../Model/AwsCryptographyPrimitivesTypes.dfy"
 
-module {:extern "AESEncryption"} AESEncryption {
```

Comment:
TODO: All the below externs changes are an internal breaking change for Java / NET (Need to move around extern).